test/gtest/cba_find2_infer.cpp

```diff
@@ -0,0 +1,184 @@
+    if(!solv.IsApplicable(fusion_ctx, fusion_problem))
+    {
+        test_skipped = true;
+        GTEST_SKIP() << solv.SolverDbId() << " Not Applicable" << conv_config;
+    }
+    ASSERT_TRUE(solv.IsApplicable(fusion_ctx, fusion_problem));
+    auto sol = solv.GetSolution(fusion_ctx, fusion_problem);
+    ASSERT_TRUE(sol.Succeeded());
+    ASSERT_TRUE(sol.invoker_factory);
+    const auto invoker = handle.PrepareInvoker(*sol.invoker_factory, sol.construction_params);
+    (invoker)(handle, invoke_ctx);
+    handle.Finish();
+}
+
+template <typename Solver>
+void RunTunableSolver(miopen::FusedProblem& problem,
+                      const miopen::AnyInvokeParams& invoke_ctx,
+                      const ConvTestCase& conv_config,
+                      bool& test_skipped)
+{
+    auto& handle = get_handle();
+    Solver solv{};
+    const auto plan           = problem.AsFusionPlan();
+    const auto fusion_problem = miopen::FusionDescription{&plan};
+    auto fusion_ctx           = miopen::FusionContext{handle};
+    if(!solv.IsApplicable(fusion_ctx, fusion_problem))
+    {
+        test_skipped = true;
+        GTEST_SKIP() << solv.SolverDbId() << " Not Applicable" << conv_config;
+    }
+    ASSERT_TRUE(solv.IsApplicable(fusion_ctx, fusion_problem));
+    auto sol = solv.GetSolution(
+        fusion_ctx, fusion_problem, solv.GetDefaultPerformanceConfig(fusion_ctx, fusion_problem));
+    ASSERT_TRUE(sol.Succeeded());
+    ASSERT_TRUE(sol.invoker_factory);
+    const auto invoker = handle.PrepareInvoker(*sol.invoker_factory, sol.construction_params);
+    (invoker)(handle, invoke_ctx);
+    handle.Finish();
+}
+
+TEST_P(ConvBiasActivFind2InferTestFloat, ConvBiasActivAsm1x1UFind2Float)
+{
+    RunTunableSolver<miopen::solver::fusion::ConvBiasActivAsm1x1U>(
+        fused_problem, invoke_params, conv_config, test_skipped);
+}
+TEST_P(ConvBiasActivFind2InferTestFloat, ConvOclDirectFwdFind2Fused)
+{
+    RunTunableSolver<miopen::solver::fusion::ConvOclDirectFwdFused>(
+        fused_problem, invoke_params, conv_config, test_skipped);
+}
+TEST_P(ConvBiasActivFind2InferTestFloat, ConvBinWinogradRxSFind2Fused)
+{
+    RunSolver<miopen::solver::fusion::ConvBinWinogradRxSFused>(
+        fused_problem, invoke_params, conv_config, test_skipped);
+}
+TEST_P(ConvBiasActivFind2InferTestFloat, ConvBinWinogradRxSf2x3g1Find2Fused)
+{
+    RunSolver<miopen::solver::fusion::ConvBinWinogradRxSf2x3g1Fused>(
+        fused_problem, invoke_params, conv_config, test_skipped);
+}
+
+TEST_P(ConvBiasActivFind2InferTestHalf, ConvCKIgemmFwdBiasActivFind2Fused)
+{
+    RunTunableSolver<miopen::solver::fusion::ConvCKIgemmFwdBiasActivFused>(
+        fused_problem, invoke_params, conv_config, test_skipped);
+}
+
+#if MIOPEN_BACKEND_HIP
+TEST_P(ConvBiasActivFind2InferTestFloatFusionFind, ConvBiasActivFind2Float_testFind)
+{
+    setEnvironmentVariable("MIOPEN_FIND_ENFORCE", "SEARCH_DB_UPDATE");
+    setEnvironmentVariable("MIOPEN_DEBUG_TUNING_ITERATIONS_MAX", "5");
```

Comment:
Can this be enforced by using the context instead? We are working on making our tests packageable, which would require merging all test targets into, resulting in a single binary, which would break this logic since env vars cannot be set and unset in the same binary.

Reply:
Or perhaps use the equivalent Find 2.0 construct

Reply:
Yes. This is 1 to 1 copy of another test with API swapped. I will think on how to set it through the internals and implement today.

Reply:
done